rskj-core/src/main/java/co/rsk/peg/union/UnionBridgeStorageProviderImpl.java

```diff
@@ -44,6 +45,14 @@ public void save(ActivationConfig.ForBlock activations) {
                 BridgeSerializationUtils::serializeCoin
             );
         }
+
+        if (!isNull(weiTransferredToUnionBridge)) {
```

Comment:
I think that using `if (nonNull(weiTransferredToUnionBridge)) {` is better here since you are already using that package.